evals/elsuite/modelgraded/classify.py

```diff
@@ -34,149 +28,119 @@ def __init__(
         modelgraded_spec: str,
         registry: Registry,
         *args,
-        match_fn: str = "starts_or_endswith",
         max_tokens: int = 1024,
         multicomp_n: Union[int, str] = 1,
         multicomp_temperature: float = 0.4,
         samples_renamings: Optional[dict[str, str]] = None,
         eval_type: Optional[str] = None,
         metaeval: bool = False,
-        eval_completion_fn: Optional[str] = None,
         modelgraded_spec_args: Optional[dict[str, dict[str, str]]] = None,
         **kwargs,
     ):
         super().__init__(completion_fns, *args, **kwargs)
+        if len(self.completion_fns) == 1:
+            self.eval_completion_fn = OpenAIChatCompletionFn(model="gpt-3.5-turbo")
```

Comment:
This is breaking unit tests where we use a call to a dummy completionfn, could we make the default always the last completion_fn in the provided arguments?

Example of breaking unit test: https://github.com/openai/evals/actions/runs/4793254818/jobs/8566241715?pr=544

Reply:
fixed! https://github.com/openai/evals/pull/828